docs/tables/azure_compute_disk_metric_read_ops_hourly.md

```diff
@@ -0,0 +1,40 @@
+# Table: azure_compute_disk_metric_read_ops_hourly
+
+GCP Monitoring Metrics provide data about the performance of your systems. The `azure_compute_disk_metric_read_ops_hourly` table provides metric statistics at 60 minute intervals for the most recent 1 year.
```

Comment:
```suggestion
GCP Monitoring Metrics provide data about the performance of your systems. The `azure_compute_disk_metric_read_ops_hourly` table provides metric statistics at 60 minute intervals for the most recent 60 days.
```

Reply:
@Subhajit97 can I know why 60 days? we are fetching the details for 1 year

Reply:
@ParthaI We should follow the below

- If interval is 24hrs, then table will store records for 1 year, and,
- if interval is 1hr, then table will store records for 60 days, and,
- if interval is 5mins, then table will store records for 5 days